typescript/lib/config.ts

```diff
@@ -71,7 +71,7 @@ export class AIConfigRuntime implements AIConfig {
   prompts: PromptWithOutputs[];
 
   filePath?: string;
-  callbackManager: CallbackManager = new CallbackManager([]);
+  callbackManager: CallbackManager = CallbackManager.createDefaultManager();
```

Comment:
Hmm, I still think having this optional and default undefined is best. It seems a bit aggressive to default logging to a specific file -- there could be issues with file permissions, adding log files by default to git commits, etc. that would make this a bad default experience; we should just make it easy to add the logging instead (e.g. they can just call CallbackManager.createWithLogging()` and pass through the constructor if they want a CM that logs to file

Reply:
I can see why default logging could seem a bit aggressive. @saqadri wdyt?

Reply:
adding a todo to linear task to revisit specifics on whether or not to include defaults